Enable even_odd_versions in bot configuration

danyeaw opened the pull request:
This PR enables the `even_odd_versions` setting in the bot section of conda-forge.yml to prevent automated updates to unstable versions.

danyeaw committed:
Enable even_odd_versions in bot configuration

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found some lint.

Here's what I've got...


For **recipe/meta.yaml**:

 * ❌ In conda-forge.yml: [`$.bot.version_updates`](https://conda-forge.org/docs/maintainer/conda_forge_yml/#bot) `=` `{'even_odd_versions': True}`.
    > {'even_odd_versions': True} is not valid under any of the given schemas
    <details>
    <summary>Schema</summary>

    ```json
    {
      "anyOf": [
        {
          "$ref": "#/$defs/BotConfigVersionUpdates"
        },
        {
          "type": "null"
        }
      ]
    }
    ```

    </details>


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/17389800200](https://github.com/conda-forge/conda-forge-webservices/actions/runs/17389800200). Examine the logs at this URL for more detail.</sub>

danyeaw commented:
@conda-forge-admin please lint

danyeaw commented:
@conda-forge-admin please lint

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

ryanvolz committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.23.15.25.29

Other tools:
- conda-build 25.7.0
- rattler-build 0.46.0
- rattler-build-conda-compat 1.4.6

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ The recipe is not parsable by parser `conda-recipe-manager`. The recipe can only be automatically migrated to the new v1 format if it is parseable by conda-recipe-manager.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/18787335947](https://github.com/conda-forge/conda-forge-webservices/actions/runs/18787335947). Examine the logs at this URL for more detail.</sub>

ryanvolz committed:
Add C++ compiler dependency b/c glib needs type_traits header (C++)

At least on macOS

ryanvolz committed:
Bump build number

ryanvolz committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.24.11.58.28

Other tools:
- conda-build 25.7.0
- rattler-build 0.46.0
- rattler-build-conda-compat 1.4.6

conda-forge-admin commented:
Hi! This is the friendly conda-forge automerge bot!

I considered the following status checks when analyzing this PR:
 - **linter**: passed
 - **azure**: passed


Thus the PR was passing and merged! Have a great day!
